docker/setup/setup.sh

```diff
@@ -159,3 +159,7 @@ RESOURCE_GROUP=${KUBERNETES_RESOURCE_GROUP_NAME}
 HELM_NAME=${helmname}
 APP_IP=${ingressip}
 EOF
+
+now=$(date +"%Y-%m-%d-%H-%M")
```

Comment:
Just one minor nit: it would be better to have the container name in a variable here instead of just the date so that the `secrets_` prefix doesn't have to be repeated below.